django_dbq/management/commands/queue_depth.py

```diff
@@ -0,0 +1,15 @@
+from django.core.management.base import BaseCommand
+from django_dbq.models import Job
+
+
+class Command(BaseCommand):
+
+    help = "Print the current depth of the given queue"
+
+    def add_arguments(self, parser):
+        parser.add_argument('queue_name', nargs='?', default='default', type=str)
```

Comment:
Why is the number of args here undefined? This'll allow multiple queue names, which may do weird things to the logging and querying

Reply:
I just copied it from the `worker` command! If I do `python manage.py foo bar` it says `error: unrecognized arguments: bar`